ui/component/settingWalletServer/view.jsx

```diff
@@ -0,0 +1,174 @@
+// @flow
+
+import React, { useState, useEffect } from 'react';
+import { FormField } from 'component/common/form';
+import ServerInputRow from './internal/inputRow';
+import ServerDisplayRow from './internal/displayRow';
+
+type DaemonSettings = {
+  lbryum_servers: Array<>
+};
+
+type StatusOfServer = {
+  host: string,
+  port: string,
+  availability: boolean,
+  latency: number,
+}
+
+type ServerInConfig = Array<string> // ['host', 'port']
+
+type DisplayOfServer = {
+  host: string,
+  port: string,
+  availability: boolean,
+}
+
+type ServerStatus = Array<StatusOfServer>
+type ServerConfig = Array<ServerInConfig>
+type DisplayList = Array<DisplayOfServer>
+
+type Props = {
+  daemonSettings: DaemonSettings,
+  getDaemonStatus: () => void,
+  setWalletServers: any => void,
+  clearWalletServers: () => void,
+  customServers: ServerConfig,
+  saveServers: string => void,
+  fetchDaemonSettings: () => void,
+  serverPrefs: ServerConfig,
+};
+
+function SettingWalletServer(props: Props) {
+  const {
+    daemonSettings,
+    fetchDaemonSettings,
+    setWalletServers,
+    getDaemonStatus,
+    clearWalletServers,
+    saveServers,
+    customServers,
+    serverPrefs,
+  } = props;
+
+  const [custom, setCustom] = useState(false);
+  const [status, setStatus] = useState([]);
```

Comment:
This is called `status` but you are calling `setStatus` with a list of wallet servers

Reply:
daemonStatus is now in the store next to daemonSettings.
now the effect calls getDaemonStatus() and status comes in as a prop and activeServers is destructured from it.